python/cuml/dask/ensemble/base.py

```diff
@@ -112,6 +117,17 @@ def _fit(self, model, dataset, convert_dtype):
                     workers=[worker],
                     pure=False)
             )
+        if len(self.workers) > len(self.active_workers):
+            if self.ignore_empty_partitions:
+                warn_text = "Data was not split among all workers"\
+                            " using only %d workers to fit" %\
+                            (len(self.active_workers))
+                warnings.warn(warn_text)
+            else:
+                raise RuntimeError("Data was not split among all workers. "

```

Comment:
should probably raise a `ValueError` instead since this relates to an incorrect value being passed in.